Gems/EMotionFX/Code/EMotionFX/Source/Importer/Importer.cpp

```diff
@@ -544,9 +544,12 @@ namespace EMotionFX
         }
 
         EMotionFX::MotionSet* motionSet = EMotionFX::MotionSet::LoadFromBuffer(memoryStart, lengthInBytes, context);
-        if (settings)
+        if (motionSet)
```

Comment:
nit, this could probably be if(motionSet && settings) to avoid the deeper nesting

Reply:
I deliberately left it like this, just in case anyone had any logging or trace like messages that maybe useful to add. It also matches the code style in the method above.